test/dynamo/test_misc.py

```diff
@@ -2936,6 +2936,105 @@ def f(pred, x):
         b = opt_fn(torch.tensor(True), torch.tensor([0.25, 0.25]))
         self.assertTrue(same(torch.sin(torch.tensor([0.25, 0.25])), b))
 
+    def test_nonzero_static(self):
```

Comment:
It seems like I can not call `torch.nonzero_static_out()`. @ezyang Is the out variant hidden from users, i.e. not accessible directly? I'm asking because I'm about to test the resizing logic here but realized that out variant is not accessible.

Reply:
You use the nonzero_static name and pass an out= kwarg